src/allmydata/test/test_client.py

```diff
@@ -75,13 +75,15 @@
 
 SOME_FURL = b"pb://abcde@nowhere/fake"
```

Comment:
Except now I reached this diff. Seems like you ought to be able to do this without making everything byte strings (mode="w"?).